pr_testing/run-pr-relvals.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -ex
+function is_in_array() {
+    local value="$1"
+    shift
+    local array=("$@")
+
+    for item in "${array[@]}"; do
+        if [[ "$item" == "$value" ]]; then
+            return 0  # Found match
+        fi
+    done
+    return 1  # No match
+}
```

Comment:
@iarspider , better to move this function in `cms-bot/_helper_functions.sh` file. That file is sources by both `pr_testing/run-pr-relvals.sh`  and `test_multiple_prs.sh`